go/vt/schemamanager/tablet_executor_test.go

```diff
@@ -54,30 +50,32 @@ func TestTabletExecutorOpen(t *testing.T) {
 	}
 }
 
-func TestTabletExecutorOpenWithEmptyPrimaryAlias(t *testing.T) {
-	ctx := context.Background()
-	ts := memorytopo.NewServer("test_cell")
-	wr := wrangler.New(logutil.NewConsoleLogger(), ts, newFakeTabletManagerClient())
-	tablet := &topodatapb.Tablet{
-		Alias: &topodatapb.TabletAlias{
-			Cell: "test_cell",
-			Uid:  1,
-		},
-		Keyspace: "test_keyspace",
-		Shard:    "0",
-		Type:     topodatapb.TabletType_REPLICA,
-	}
-	// This will create the Keyspace, Shard and Tablet record.
-	// Since this is a replica tablet, the Shard will have no primary.
-	if err := wr.InitTablet(ctx, tablet, false /*allowPrimaryOverride*/, true /*createShardAndKeyspace*/, false /*allowUpdate*/); err != nil {
-		t.Fatalf("InitTablet failed: %v", err)
-	}
-	executor := NewTabletExecutor("TestTabletExecutorOpenWithEmptyPrimaryAlias", wr, testWaitReplicasTimeout)
-	if err := executor.Open(ctx, "test_keyspace"); err == nil || !strings.Contains(err.Error(), "does not have a primary") {
-		t.Fatalf("executor.Open() = '%v', want error", err)
-	}
-	executor.Close()
-}
+// TODO: had this comment this out because it was causing an import cycle.
+// need to rewrite this test which depends on InitTablet in wrangler.
+//func TestTabletExecutorOpenWithEmptyPrimaryAlias(t *testing.T) {
+//	ctx := context.Background()
+//	ts := memorytopo.NewServer("test_cell")
+//	wr := wrangler.New(logutil.NewConsoleLogger(), ts, newFakeTabletManagerClient())
+//	tablet := &topodatapb.Tablet{
+//		Alias: &topodatapb.TabletAlias{
+//			Cell: "test_cell",
+//			Uid:  1,
+//		},
+//		Keyspace: "test_keyspace",
+//		Shard:    "0",
+//		Type:     topodatapb.TabletType_REPLICA,
+//	}
+//	// This will create the Keyspace, Shard and Tablet record.
+//	// Since this is a replica tablet, the Shard will have no primary.
+//	if err := wr.InitTablet(ctx, tablet, false /*allowPrimaryOverride*/, true /*createShardAndKeyspace*/, false /*allowUpdate*/); err != nil {
+//		t.Fatalf("InitTablet failed: %v", err)
+//	}
+//	executor := NewTabletExecutor("TestTabletExecutorOpenWithEmptyPrimaryAlias", ts, newFakeTabletManagerClient(), logutil.NewConsoleLogger(), testWaitReplicasTimeout)
+//	if err := executor.Open(ctx, "test_keyspace"); err == nil || !strings.Contains(err.Error(), "does not have a primary") {
+//		t.Fatalf("executor.Open() = '%v', want error", err)
+//	}
+//	executor.Close()
+//}
```

Comment:
Any reason this commented code should not be removed?

Reply:
This is fixed in https://github.com/vitessio/vitess/pull/9537/commits/4b37d644f3d3cb98b7d5c3df65ea8f6708652fc3